src/components/views/elements/SettingsFlag.js

```diff
@@ -34,12 +34,13 @@ module.exports = createReactClass({
 
     getInitialState: function() {
         return {
-            value: SettingsStore.getValueAt(
+            // convert to Boolean to protect against null-capable "tri-state" Settings e.g fallbackICEServerAllowed
```

Comment:
This is an indicator of using the wrong component :(

Reply:
Yes but for the exact case here I believe that is fine, for others it may not be

Reply:
It's not a good idea for us to be doing this. We should use the right component.